.github/workflows/integration.yml

```diff
@@ -95,32 +88,30 @@ jobs:
         uses: 'actions/setup-go@4d34df0c2316fe8122ab82dc22947d607c0c91f9' # ratchet:actions/setup-go@v4
         with:
           go-version: '1.20'
-      - name: 'Build images'
-        run: ./scripts/build.sh
-    #   - id: 'auth'
-    #     name: 'Authenticate to Google Cloud'
-    #     uses: 'google-github-actions/auth@ef5d53e30bbcd8d0836f4288f5e50ff3e086997d' # ratchet:google-github-actions/auth@v1
-    #     with:
-    #       workload_identity_provider: '${{ env.WIF_PROVIDER }}'
-    #       service_account: '${{ env.WIF_SERVICE_ACCOUNT }}'
-    #       token_format: 'access_token'
-    #   - name: 'Authenticate to Artifact Registry'
-    #     uses: 'docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a' # ratchet:docker/login-action@v2
-    #     with:
-    #       username: 'oauth2accesstoken'
-    #       password: '${{ steps.auth.outputs.access_token }}'
-    #       registry: '${{ env.REGISTRY_HOST }}'
-    #   # goreleaser requires a tag to publish images to container registry.
-    #   # We create a local tag to make it happy.
-    #   - run: |-
-    #       git config user.name "${GITHUB_ACTOR}"
-    #       git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
-    #       git tag -f `date "+%Y%m%d%H%M%S"`
-    #   - name: 'Build the service containers and push to the registry with goreleaser'
-    #     uses: 'goreleaser/goreleaser-action@f82d6c1c344bcacabba2c841718984797f664a6b' # ratchet:goreleaser/goreleaser-action@v4
-    #     with:
-    #       version: 'v1.15.2' # Manually pinned
-    #       args: 'release -f .goreleaser.docker.yaml --rm-dist --skip-validate'
+      - id: 'auth'
+        name: 'Authenticate to Google Cloud'
+        uses: 'google-github-actions/auth@ef5d53e30bbcd8d0836f4288f5e50ff3e086997d' # ratchet:google-github-actions/auth@v1
+        with:
+          workload_identity_provider: '${{ env.WIF_PROVIDER }}'
+          service_account: '${{ env.WIF_SERVICE_ACCOUNT }}'
+          token_format: 'access_token'
+      - name: 'Authenticate to Artifact Registry'
+        uses: 'docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a' # ratchet:docker/login-action@v2
+        with:
+          username: 'oauth2accesstoken'
+          password: '${{ steps.auth.outputs.access_token }}'
+          registry: '${{ env.REGISTRY_HOST }}'
+      # goreleaser requires a tag to publish images to container registry.
+      # We create a local tag to make it happy.
+      - run: |-
+          git config user.name "${GITHUB_ACTOR}"
+          git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+          git tag -f `date "+%Y%m%d%H%M%S"`
+      - name: 'Build the service containers and push to the registry with goreleaser'
+        uses: 'goreleaser/goreleaser-action@f82d6c1c344bcacabba2c841718984797f664a6b' # ratchet:goreleaser/goreleaser-action@v4
+        with:
+          version: 'v1.15.2' # Manually pinned
```

Comment:
Can you use the updated version like this: https://github.com/abcxyz/pmap/blob/4fb2dc18cf4421d5e826b5070102188273a03f21/.github/workflows/ci.yml#L89

Reply:
Ack. updated to newest version.